restcomm.android.client.sdk/src/main/java/org/mobicents/restcomm/android/client/sdk/RCConnection.java

```diff
@@ -560,10 +563,19 @@ void initializeWebrtc(boolean videoEnabled)
         for (String permission : MANDATORY_PERMISSIONS) {
             if (context.checkCallingOrSelfPermission(permission) != PackageManager.PERMISSION_GRANTED) {
                 logAndToast("Permission " + permission + " is not granted");
-                // TODO: return error to RCConnection listener
+                listener.onPermissionNotGranted(this, permission);
```

Comment:
OK. What I meant is this use case:
1. do not grant the mandatory permissions to the app
2. try to make outbound call - this results in an error and onDisconnected() is called
3. try to make outbound call (again) - because device state is BUSY, this results in null returned from connect()
So IMHO we have 2 different behaviors for the same case.
What do you think.